products/bash/deploy-og-sub.sh

```diff
@@ -29,9 +30,13 @@ function usage() {
 }
 
 namespace="cp4i"
+DEPLOY_DEMOS=false
 
-while getopts "n:p" opt; do
+while getopts "n:d" opt; do
   case ${opt} in
+  p)
+    DEPLOY_DEMOS=true
```

Comment:
Where is this actually used? It seems to be set in this script but I can't see anywhere it's used?

Reply:
Should be used below:
```
if [[ "${DEPLOY_DEMOS}" == "true" ]]; then
```
should be:
```
if [[ "${USE_PRERELEASE_CATALOGS}" == "true" ]]; then
```